docs/src/api/methods/_getBlockHeight.mdx

```diff
@@ -0,0 +1,72 @@
+import {
+  DocBlock,
+  DocSideBySide,
+  CodeParams,
+  Parameter,
+  Field,
+  Values,
+  CodeSnippets,
+} from "../../../components/CodeDocBlock";
+
+<DocBlock>
+
+## getBlockHeight
+
+Returns the current block height of the node
+
+<DocSideBySide>
+
+<CodeParams>
+
+### Parameters:
+
+<Parameter type={"object"} optional={true}> 
+Configuration object containing the following fields:
+
+<Field
+  name="commitment"
+  type="string"
+  optional={true}
+  href="/api#configuring-state-commitment"
```

Comment:
All of the `commitment` links need to point to `/api/http#configuring-state-commitment`